numba/tests/test_parfors.py

```diff
@@ -1263,6 +1264,11 @@ def ddot(a, v):
                "hardware")
         self.assertIn(msg, str(raised.exception))
 
+@register_jitable
+def issue7854_proc(u, i, even, size):
+    for j in range((even + i + 1)%2 +1, size-1, 2):
+        u[i, j] = u[i, j+1] + u[i, j-1] + u[i+1,j ] + u[i-1, j] + 1
```

Comment:
Can this expression be made smaller and have the same effect?

Reply:
Done.